[google-cloud-cpp] Build failure

Package: google-cloud-cpp[core,storagetransfer]:arm64-osx -> 2.11.0

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: f6a5d4e8e 2023-06-20 (16 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks --allow-unsupported google-cloud-cpp[core,storagetransfer]`

**Failure logs**

```
-- Downloading https://github.com/googleapis/google-cloud-cpp/archive/v2.11.0.tar.gz -> googleapis-google-cloud-cpp-v2.11.0.tar.gz...
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/googleapis-google-cloud-cpp-v2.11.0.tar.gz
-- Applying patch support_absl_cxx17.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean
-- Found external ninja('1.11.1').
-- Configuring arm64-osx-dbg
-- Configuring arm64-osx-rel
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/google-cloud-cpp/portfile.cmake:49 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/install-arm64-osx-dbg-out.log</summary>

```
[1/6085] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -Dcfgdir= -P /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/tmp/googleapis_download-mkdirs.cmake && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -E touch /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/src/googleapis_download-stamp/googleapis_download-mkdir
[2/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/build_info.cc.o -MF google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/build_info.cc.o.d -o google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/build_info.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/internal/build_info.cc
[3/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/api_client_header.cc.o -MF google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/api_client_header.cc.o.d -o google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/api_client_header.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/internal/api_client_header.cc
[4/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/compute_engine_util.cc.o -MF google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/compute_engine_util.cc.o.d -o google/cloud/CMakeFiles/google_cloud_cpp_common.dir/internal/compute_engine_util.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/internal/compute_engine_util.cc
...
Skipped 4989 lines
...

[4888/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights_protos.dir/google/cloud/storageinsights/v1/storageinsights.grpc.pb.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights_protos.dir/google/cloud/storageinsights/v1/storageinsights.grpc.pb.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights_protos.dir/google/cloud/storageinsights/v1/storageinsights.grpc.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights/google/cloud/storageinsights/v1/storageinsights.grpc.pb.cc
[4889/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_metadata_decorator.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_metadata_decorator.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_metadata_decorator.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_metadata_decorator.cc
[4890/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_option_defaults.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_option_defaults.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_option_defaults.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_option_defaults.cc
[4891/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_logging_decorator.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_logging_decorator.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_logging_decorator.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_logging_decorator.cc
[4892/6085] : && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f google/cloud/storageinsights/libgoogle_cloud_cpp_storageinsights_protos.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc google/cloud/storageinsights/libgoogle_cloud_cpp_storageinsights_protos.a  google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights_protos.dir/google/cloud/storageinsights/v1/storageinsights.pb.cc.o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights_protos.dir/google/cloud/storageinsights/v1/storageinsights.grpc.pb.cc.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib google/cloud/storageinsights/libgoogle_cloud_cpp_storageinsights_protos.a && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -E touch google/cloud/storageinsights/libgoogle_cloud_cpp_storageinsights_protos.a && :
[4893/6085] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/support && /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/protobuf/protoc-3.21.12.0 --cpp_out /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/support --proto_path /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/src/googleapis_download --proto_path /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/src/googleapis_download/google/cloud/support/v2/actor.proto
[4894/6085] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer && /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/protobuf/protoc-3.21.12.0 --plugin=protoc-gen-grpc=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/grpc/grpc_cpp_plugin --grpc_out=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer --cpp_out=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer --proto_path /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/src/googleapis_download --proto_path /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/src/googleapis_download/google/storagetransfer/v1/transfer_types.proto
[4895/6085] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/support && /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/protobuf/protoc-3.21.12.0 --cpp_out /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/support --proto_path /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/src/googleapis_download --proto_path /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis/src/googleapis_download/google/cloud/support/v2/attachment.proto
[4896/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_stub.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_stub.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_stub.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_stub.cc
[4897/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_tracing_connection.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_tracing_connection.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_tracing_connection.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_tracing_connection.cc
[4898/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_connection_impl.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_connection_impl.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_connection_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_connection_impl.cc
[4899/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_stub_factory.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_stub_factory.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_stub_factory.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_stub_factory.cc
[4900/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.cc
FAILED: google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.pb.cc.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.cc
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.cc:4:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.h:4201:24: error: expected member name or ';' after declaration specifiers
  static constexpr GID GID_MAX =
  ~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/syslimits.h:86:28: note: expanded from macro 'GID_MAX'
#define GID_MAX            2147483647U  /* max value for a gid_t (2^31-2) */
                           ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.cc:1263:48: error: expected unqualified-id
constexpr MetadataOptions_GID MetadataOptions::GID_MAX;
                                               ^
2 errors generated.
[4901/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.pb.cc
FAILED: google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.pb.cc.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.pb.cc
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.pb.cc:4:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.pb.h:39:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.h:4201:24: error: expected member name or ';' after declaration specifiers
  static constexpr GID GID_MAX =
  ~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/syslimits.h:86:28: note: expanded from macro 'GID_MAX'
#define GID_MAX            2147483647U  /* max value for a gid_t (2^31-2) */
                           ^
1 error generated.
[4902/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_tracing_stub.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_tracing_stub.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/internal/storage_insights_tracing_stub.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/internal/storage_insights_tracing_stub.cc
[4903/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_connection_idempotency_policy.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_connection_idempotency_policy.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_connection_idempotency_policy.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/storage_insights_connection_idempotency_policy.cc
[4904/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_client.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_client.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_client.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/storage_insights_client.cc
[4905/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/speech/CMakeFiles/google_cloud_cpp_speech.dir/v2/internal/speech_connection_impl.cc.o -MF google/cloud/speech/CMakeFiles/google_cloud_cpp_speech.dir/v2/internal/speech_connection_impl.cc.o.d -o google/cloud/speech/CMakeFiles/google_cloud_cpp_speech.dir/v2/internal/speech_connection_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/speech/v2/internal/speech_connection_impl.cc
[4906/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.grpc.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.grpc.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.grpc.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.grpc.pb.cc
FAILED: google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.grpc.pb.cc.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.grpc.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.grpc.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer_types.grpc.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.grpc.pb.cc
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.grpc.pb.cc:5:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.h:4201:24: error: expected member name or ';' after declaration specifiers
  static constexpr GID GID_MAX =
  ~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/syslimits.h:86:28: note: expanded from macro 'GID_MAX'
#define GID_MAX            2147483647U  /* max value for a gid_t (2^31-2) */
                           ^
1 error generated.
[4907/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storageinsights -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_connection.cc.o -MF google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_connection.cc.o.d -o google/cloud/storageinsights/CMakeFiles/google_cloud_cpp_storageinsights.dir/v1/storage_insights_connection.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storageinsights/v1/storage_insights_connection.cc
[4908/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.grpc.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.grpc.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.grpc.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.grpc.pb.cc
FAILED: google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.grpc.pb.cc.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.grpc.pb.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.grpc.pb.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer_protos.dir/google/storagetransfer/v1/transfer.grpc.pb.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.grpc.pb.cc
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.grpc.pb.cc:5:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.pb.h:39:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.h:4201:24: error: expected member name or ';' after declaration specifiers
  static constexpr GID GID_MAX =
  ~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/syslimits.h:86:28: note: expanded from macro 'GID_MAX'
#define GID_MAX            2147483647U  /* max value for a gid_t (2^31-2) */
                           ^
1 error generated.
[4909/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_auth_decorator.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_auth_decorator.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_auth_decorator.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_auth_decorator.cc
FAILED: google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_auth_decorator.cc.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_auth_decorator.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_auth_decorator.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_auth_decorator.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_auth_decorator.cc
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_auth_decorator.cc:19:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_auth_decorator.h:22:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_stub.h:27:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.grpc.pb.h:22:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.pb.h:39:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.h:4201:24: error: expected member name or ';' after declaration specifiers
  static constexpr GID GID_MAX =
  ~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/syslimits.h:86:28: note: expanded from macro 'GID_MAX'
#define GID_MAX            2147483647U  /* max value for a gid_t (2^31-2) */
                           ^
1 error generated.
[4910/6085] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_connection_impl.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_connection_impl.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_connection_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_connection_impl.cc
FAILED: google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_connection_impl.cc.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCARES_STATICLIB -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/external/googleapis -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wconversion -Wno-sign-conversion -std=gnu++14 -MD -MT google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_connection_impl.cc.o -MF google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_connection_impl.cc.o.d -o google/cloud/storagetransfer/CMakeFiles/google_cloud_cpp_storagetransfer.dir/v1/internal/storage_transfer_connection_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_connection_impl.cc
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_connection_impl.cc:19:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_connection_impl.h:23:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/src/v2.11.0-46eb30c86b.clean/google/cloud/storagetransfer/v1/internal/storage_transfer_stub.h:27:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.grpc.pb.h:22:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer.pb.h:39:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/google-cloud-cpp/arm64-osx-dbg/google/cloud/storagetransfer/google/storagetransfer/v1/transfer_types.pb.h:4201:24: error: expected member name or ';' after declaration specifiers
  static constexpr GID GID_MAX =
  ~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/syslimits.h:86:28: note: expanded from macro 'GID_MAX'
#define GID_MAX            2147483647U  /* max value for a gid_t (2^31-2) */
                           ^
1 error generated.
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "gstreamer",
      "default-features": false,
      "features": [
        "gpl"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/google-cloud-cpp/portfile.cmake
+++ b/ports/google-cloud-cpp/portfile.cmake
@@ -3,8 +3,8 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO googleapis/google-cloud-cpp
-    REF v${VERSION}
-    SHA512 d8515906b81e5bb8babe76bdbcba9462eeb9906e7099772d47d871483fa26fbc86943c41c1a916003c0bce767d15ebbbf870699ef75f2fdbcb61f9879d8b4017
+    REF "v${VERSION}"
+    SHA512 206d07eb65da472a9cc743ef6de0ddc7c0e13833cb990dc2415681be7c2a9d8ddfac6a25880ff3dbd2c73257d2ad8f7b8331cc1755e31d85bdb5386a12c91dbe
     HEAD_REF main
     PATCHES
         support_absl_cxx17.patch

--- a/ports/google-cloud-cpp/vcpkg.json
+++ b/ports/google-cloud-cpp/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "google-cloud-cpp",
-  "version": "2.11.0",
+  "version": "2.12.0",
   "description": "C++ Client Libraries for Google Cloud Platform APIs.",
   "homepage": "https://github.com/googleapis/google-cloud-cpp",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2921,7 +2921,7 @@
       "port-version": 3
     },
     "google-cloud-cpp": {
-      "baseline": "2.11.0",
+      "baseline": "2.12.0",
       "port-version": 0
     },
     "google-cloud-cpp-common": {

--- a/versions/g-/google-cloud-cpp.json
+++ b/versions/g-/google-cloud-cpp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f7179212beeb134849f381dbe2c200580cc37c97",
+      "version": "2.12.0",
+      "port-version": 0
+    },
+    {
       "git-tree": "0a4dc6845aca21d78853e4f6bc11e9bc5174b7b2",
       "version": "2.11.0",
       "port-version": 0

